recipe/meta.yaml

```
@@ -1,30 +1,32 @@
-{% set version = "4.4.0" %}
+{% set version = "5.27.0" %}
 
 package:
   name: neo4j-python-driver
   version: {{ version }}
 
 source:
   url: https://github.com/neo4j/neo4j-python-driver/archive/{{ version }}.tar.gz
-  sha256: 1c63513674b047d9cfcac950225069364af7ae1ccc504d4c696566a626fb3ed0
+  sha256: cefd4296efe564b2feb96f628699e0cbf421888ba5815a1b92b2c1b5e71ff07d
 
 build:
   number: 0
-  skip: True  # [py<36]
-  script: python -m pip install --no-deps --ignore-installed .
+  skip: True  # [py<38]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vvv
 
 requirements:
-  build:
-    - {{ compiler('c') }}
   host:
     - python
-    - cython
     - pip
     - setuptools
+    - tomlkit
     - wheel
   run:
     - python
     - pytz
+  run_constrained:
+    - pandas
+    - numpy
+    - pyarrow
```

```suggestion
    - pandas >=1.1.0,<3.0.0
    - numpy >=1.7.0 < 3.0.0
    - pyarrow >=1.0.0
```
To follow https://github.com/neo4j/neo4j-python-driver/blob/5.27.0/pyproject.toml#L57-L63